control_lvmm

```diff
@@ -0,0 +1,15 @@
+Package: labview-memory-management-tools-{labview_version}-support
```

Comment:
The NIPM convention is to have <product>-<version>-support, so
`labview-memory-management-tools-labview-{labview_version}-support`.

Reply:
Also, the package name should start with the company.

`ni-labview-memory-management-tools-labview-{labview_version}-support`